config/presets/acl.json

```
@@ -3,10 +3,33 @@
   "@graph": [
     {
       "@id": "urn:solid-server:default:AclManager",
-      "@type": "UrlBasedAclManager"
+      "@type": "UrlBasedAclManager",
+      "UrlBasedAclManager:_converter": {
+        "@id": "urn:solid-server:default:RdfToQuadConverter"
+      }
     },
+
     {
       "@id": "urn:solid-server:default:AclAuthorizer",
```

Perhaps for clarity now
```suggestion
      "@id": "urn:solid-server:default:AclBasedAuthorizer",
```